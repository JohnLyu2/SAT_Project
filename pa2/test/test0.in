a123

a+c
a*---a
a+b;a:0,b:1
 ( (-a)+(a*b)) * a * (c + -b) *-c
(a+b+c)*(a+b+-c)*(-b+a +c)*(-a*-c)
(a+b+c)*(a+b+-c)*(-b+a +c)*(a+-b+-c)*(-a+b+c)*(-a+b+-c)*(-a+-b+c)*(-a+-c+-c)
(a+b+c)*(a+b+-c)*(-b+a +c)*(a+-b+-c)*(-a+b+c)*(-a+b+-c)*(-a+-b+c)*(-a+-c+-b)
(a+b+c)*(a+b+-c)*(-b+a +c)*(a+-b+-c)*(-a+b+c)*(-a+b+-c)*(-a+-b+c)
