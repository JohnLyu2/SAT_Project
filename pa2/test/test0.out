sat
Error: invalid input
sat
unsat
Error: invalid input
unsat
unsat
unsat
unsat
sat
